theblues/charmstore.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 import requests
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, RequestException, Timeout
```

Comment:
please use code convetions of formatting

    from requests.exceptions import (
        HTTPError,
        RequestException,
        Timeout
    )